pkg/rancher-components/src/components/Pill/StatusIndicator/types.ts

```diff
@@ -0,0 +1,7 @@
+export type Shape = 'disc' | 'horizontal-bar' | 'vertical-bar';
+export type Status = 'info' | 'success' | 'warning' | 'error' | 'unknown' | 'none';
+
+export interface CardProps {
```

Comment:
`CardProps` doesn't appear to be the right name. What if we named this something like `StatusIndicatorProps`?

Reply:
Correct. I'll fix that.